.github/workflows/pr.yaml

```diff
@@ -44,6 +44,10 @@ jobs:
       - name: Test
         # TODO enable build cache, either --build-cache here, or org.gradle.caching=true in gradle.properties
         run: ./gradlew check --stacktrace
+      - name: Upload snapshot
+        uses: actions/upload-artifact@v2
+        with:
+          path: agent/agent/build/libs/applicationinsights-agent-*-SNAPSHOT.jar
```

Comment:
will there be a link available on the pr automatically?  just curious.

Reply:
yes! e.g. click into the build below, and go to "Summary" (upper left), then click on "Artifacts" (to the left)